### ISSUE TITLE
Fake graded assignment data is unrealistic

### ISSUE THREAD

**Issue by justin0022** [tl-its-umich-edu/my-learning-analytics#726]
The fake assignment grades are unrealistic - often the grades are much higher than 100%, causing a large amount of overflow. 

I can work around this by manually adjusting the grades or preventing the overflow in the first place, but just wanted to flag this for now.
<img width="1469" alt="Screen Shot 2019-09-27 at 2 10 40 PM" src="https://user-images.githubusercontent.com/8836578/65802510-bc93e680-e130-11e9-915d-4e6c1a34678b.png">

**Comment by jonespm** [comment]
Yeah I know, the depersonalization has a bug that I haven't had time to look into with assignments. We'd like to fix this and convert these to fixtures, but have been too busy getting ready still to deploy for start of semester. It's a known issue and I'll fix it.

**Comment by justin0022** [comment]
No problem, it's not a pressing issue by any means, just makes the AssignmentPlanning redesign a bit trickier!

**Comment by pushyamig** [comment]
The effort of depersonalized data is [here](https://github.com/tl-its-umich-edu/myla-depersonalizer)

@jonespm I feel the repo should be mentioned in the MyLA readme so that more depersonalized data for all views can be added here by the community if they are willing to participate

**Comment by jonespm** [comment]
I could add it, I think there's at least this issue of assignment data to resolve before anyone else uses it though.

**Comment by jennlove-um** [comment]
@justin0022 Is UBC still using the fake data file?
